.github/workflows/test-win.yml

```diff
@@ -43,15 +43,11 @@ jobs:
           dir deps\fftw3
           echo "GITHUB_WORKSPACE: $env:GITHUB_WORKSPACE"
 
-      - name: Build and test
+      - name: Build
         run: |
           "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-          cmake . -DFFTW_INCLUDE_DIRS="deps\fftw3" -DFFTW_LIBRARIES="$env:GITHUB_WORKSPACE\deps\fftw3\libfftw3-3.lib" -DPYTHON_BINDINGS=ON -DBUILD_TESTS_EXAMPLES=ON -G "Visual Studio 17 2022" -A x64
-          cmake --build . --config Release 
-          cp deps\fftw3\* tests\Release\
-          cp src\Release\* tests\Release\
-          cp bindings\c\src\Release\* tests\Release\
-          ctest -V -C Release
+          cmake . -DFFTW_INCLUDE_DIRS="deps\fftw3" -DFFTW_LIBRARIES="$env:GITHUB_WORKSPACE\deps\fftw3\libfftw3-3.lib" -DPYTHON_BINDINGS=ON -G "Visual Studio 17 2022" -A x64
```

Comment:
Do you want to create a separate job for the installer? That way windows gets actually tested?